### ISSUE TITLE
Avatar error

### ISSUE THREAD

**Issue by irsadmin** [KETSE/casebox#36]
When i upload profile picture to any account, its shows error.

**Comment by tvitalie** [comment]
Hi @irsadmin,
I suppose you miss imagick  php module.
Check logs under "casebox/log" folder.

**Comment by irsadmin** [comment]
No, i don't miss imagick php module or anything else. Because i download casebox's VMware version, which was ready to use.
